src/AutoMapper/MapperConfiguration.cs

```diff
@@ -415,4 +414,21 @@ private static LambdaExpression TryCatch(LambdaExpression mapExpression, Paramet
             }
         }
     }
-}
+
+    public struct LockingConcurrentDictionary<TKey, TValue>

```

Comment:
Useful for who? Asking because I've made a few of these internal and broke people when I've moved/changed them. Like reflection extension methods that people started using because they were public.